openmrs-orthanc/src/test/resources/response/orthanc-single-series-response.json

```diff
@@ -0,0 +1,385 @@
+{
+  "ExpectedNumberOfInstances": null,
+  "ID": "17cc7e52-4f1a3e4d-9182f727-56e9cc71-c037892f",
+  "Instances": [
+    "021587a8-3b4629df-96455d78-79c4df7b-1ea4714e",
+    "02367bc6-9a6fd565-29c521f7-f46f10eb-99b7523a",
+    "02d0fdd3-db518ddd-b90625dc-926f3ebc-d0c327fc",
+    "03ce1bf6-2f5ec727-fb3ce2e2-7a130dd8-e123d217",
```

Comment:
We don't need all of the instances here, one or two are enough since we are mocking. Makes it easily to read through.

Reply:
Done